PSScripts/Import-ApimSwaggerApiDefinition.ps1

```diff
@@ -0,0 +1,87 @@
+<#
+.SYNOPSIS
+Update an APIM API with a swagger definition
+
+.PARAMETER ResourceGroupName
+The name of the resource group that contains the APIM instnace
+
+.PARAMETER InstanceName
+The name of the APIM instnace
+
+.PARAMETER ApiName
+The name of the API to update
+
+.PARAMETER SwaggerSpecificationUrl
+The full path to the swagger defintion
+
+For example:
+https://sit-manage-vacancy.apprenticeships.sfa.bis.gov.uk/swagger/docs/v1
+
+#>
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$true)]
+    [String]$ApimResourceGroup,
+    [Parameter(Mandatory=$true)]
+    [String]$InstanceName,
+    [Parameter(Mandatory=$true)]
+    [String]$ApiName,
+    [Parameter(Mandatory=$true)]
+    [String]$SwaggerSpecificationUrl,
+    [Parameter(Mandatory=$false, ParameterSetName="File")]
+    [Switch]$SwaggerSpecificationFile,
```

Comment:
No .PARAMETER section for SwaggerSpecificationUrl or SwaggerSpecificationFile

Reply:
Added